app/models/move.rb

```diff
@@ -46,6 +46,18 @@ class Move < VersionedModel
   validates :move_type, inclusion: { in: move_types }
   validates :person, presence: true
   validates :reference, presence: true
+  validates(
+    :move_agreed,
+    inclusion: { in: [true, false] },
+    if: ->(move) { move.move_type == 'prison_transfer' },
```

Comment:
no biggee but could 'prison_transfer' be a constant ?

Reply:
I'll change this in the following PR (that expose that field in the API) if it works for you

Reply:
@alexdesi sure :+1: